docs/concepts/providers.md

```diff
@@ -7,41 +7,34 @@ indent: true
 
 # Providers
 
-Providers extend Crossplane to enable infrastructure resource provisioning. In
-order to provision a resource, a Custom Resource Definition (CRD) needs to be
-registered in your Kubernetes cluster and its controller should be watching the
-Custom Resources those CRDs define. Provider packages contain many Custom
-Resource Definitions and their controllers.
+Providers are Crossplane packages that bundle a set of [Managed
+Resources][managed-resources] and their respective controllers to allow
+Crossplane to provision the respective infrastructure resource.
 
 Here is the list of prominent providers:
 
-### AWS Provider
+|   |   |   |   |
+|---|---|---|---|
+| Provider AWS  |  [GitHub Repo](https://github.com/crossplane/provider-aws) | [API Reference](https://doc.crds.dev/github.com/crossplane/provider-aws)  |  [Amazon Web Services (AWS) IAM User] |
```

Comment:
I believe there was desire to standardized on named links instead of inline links -- I don't have a strong preference, but it might be good to be consistent

Reply:
I prefer named links, both in general and because it's consistent with what we're doing today.

Reply:
In this case, I find using named links make it actually harder to read and maintain. I'm happy to comply for consistency reasons. Is there any other reason other than consistency?

Reply:
I think other than being consistent it's just subjective.

Reply:
Fixed with https://github.com/crossplane/crossplane/pull/2816/commits/c5b9bcc33b311c7b6d82f501d3bd0fc3eda07dca